Source/Bogus/DataSet.cs

```diff
@@ -91,18 +100,18 @@ protected internal virtual BValue Get(string category, string path)
       /// <summary>
       /// Determines if a key exists in the locale.
       /// </summary>
+      /// <returns>A boolean to indicate if the locale exists.</returns>
       protected internal virtual bool HasKey(string path, bool includeFallback = true)
       {
-         if( includeFallback )
-            return Database.HasKey(this.Category, path, this.Locale);
-
-         return Database.HasKey(this.Category, path, this.Locale, null);
+         return includeFallback ? Database.HasKey(this.Category, path, this.Locale)
+                   : Database.HasKey(this.Category, path, this.Locale, null);
```

Comment:
It would be cool if this one could be reverted back to the original `if` statement. I'm not particularly a fan of log ternary operator lines that span multiple lines like this. At most, I'm okay if you wrap the original `if` bock with `{ }`.

In particular, I prefer to have the code in more complicated classes like this `DataSet` class to be slightly more on the verbose side as it helps with debugging issues since this is one of the core classes in **Bogus**.

Reply:
~~For the code Quality!~~ In all seriousness I was skeptical about this changes myself because it wasn't *that* readable so I reverted it, my apologies.